### PR TITLE
my_20240828_branch

### PR DESCRIPTION
**Pull request by abhishekabhi063**
# New Prod config

Self check-list:

- [x] I have checked the lstchain config, in particular for:
  - [x] `az_tel` instead of `sin_az_tel` if data to be analyzed have been produced with lstchain <= v0.9.7
  - [x] "increase_nsb" and "increase_psf" are provided in "image_modifier" (if used)
- [x] I have checked the environment in the lstmcpipe config and it is the one used to analyse DL>1 data
- [x] I have provided the command (in README), or script (in additionnal .py file) used to produce the lstmcpipe config

## Prod_ID

20240828_v0.10.9_GC_dec_min_2924_tuned

### CODE REVIEW

File: production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/lstmcpipe_config_2024-08-28_PathConfigAllSkyFull.yaml [hunk 1]
```diff
@@ -0,0 +1,3048 @@
+# lstmcpipe generated config from PathConfigAllSkyFull - 2024-08-28
+
+workflow_kind: lstchain
+
+# prod_id ex: local_no_n_islands. Default; v00 (if key left empty or None)
+prod_id: 20240828_v0.10.9_GC_dec_min_2924_tuned
+
+source_environment:
+  source_file: /fefs/aswg/software/conda/etc/profile.d/conda.sh
+  conda_env: lstchain-v0.10.7
```
Comment (vuillaut) [inline production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/lstmcpipe_config_2024-08-28_PathConfigAllSkyFull.yaml:10]: this needs to be changed to lstchain-v0.10.9

Comment (abhishekabhi063) [reply production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/lstmcpipe_config_2024-08-28_PathConfigAllSkyFull.yaml]: I have updated the config file by changing the lstchain environment

File: production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Prod 20240828_v0.10.9_GC_dec_min_2924_tuned
+
+## Description of the config
+
```
Comment (vuillaut) [inline production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md:4]: If you could also provide the command you used to produce the config - it is very useful to other users / to reproduce / to debug, as well as a short description of why this production was requested.
Thanks.

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md (hunk 2 above)
Comment (abhishekabhi063) [reply production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md:4]: I have added commands for producing lstchain config and lstmcpipe config along with a short desciption of why this production is being requested.

Comment (vuillaut) [reply production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md:4]: Thanks !

**Review by vuillaut — APPROVED** [review]
LGTM

**Pull request merged**